react/components/Input/index.js

```diff
@@ -71,6 +84,8 @@ class Input extends Component {
         'You should not use both prefix and suffix props in the same input. '
       )
     }
+
+    this.handleAutoFocus()
```

Comment:
shouldn't you add something like this in `componentDidUpdate` as well?

Reply:
yep. I'll do that.